tasks/main.yml

```diff
@@ -28,5 +28,10 @@
   notify:
     - Restart Zookeeper
 
+- name: Configure SERVER_JVMFLAGS
+  lineinfile: dest=/etc/default/zookeeper regexp="SERVER_JVMFLAGS" line="SERVER_JVMFLAGS=\"{{ zookeeper_server_jvmflags }}\""
```

Comment:
This does not appear to be taking effect because of a missing `export`:

**Before**

```
# export SERVER_JVMFLAGS=""
```

**After**

```
SERVER_JVMFLAGS="-Xmx2048m"
```

In addition, it would be good to have each argument on a new line to make reading this long task easier:

```yaml
- name: Configure SERVER_JVMFLAGS
  lineinfile: dest=/etc/default/zookeeper
              regexp="SERVER_JVMFLAGS"
              line="SERVER_JVMFLAGS=\"{{ zookeeper_server_jvmflags }}\""
  notify:
    - Restart Zookeeper
```